test/controllers/OpenPaymentsControllerSpec.scala

```diff
@@ -69,57 +67,68 @@ class OpenPaymentsControllerSpec extends ControllerBaseSpec {
+  "Calling the openPayments action" when {
 
     "the user has open payments" should {
 
-      "return the payments view" in new Test {
+      "return 200 (OK)" in new Test {
         override def setupMocks(): Unit = {
           super.setupMocks()
           (mockPaymentsService.getOpenPayments(_: String)(_: HeaderCarrier, _: ExecutionContext))
             .expects(*, *, *)
             .returns(Future.successful(Some(Payments(Seq(payment, payment)))))
         }
+        private val result = target.openPayments()(fakeRequest)
 
-        val result: Result = await(target.openPayments()(fakeRequest))
-        val document: Document = Jsoup.parse(bodyOf(result))
-
-        document.select("h1").first().text() shouldBe "What you owe"
+        status(result) shouldBe Status.OK
       }
     }
 
     "the user has no open payments" should {
 
-      "return the no payments view" in new Test {
+      "return 200 (OK)" in new Test {
         override def setupMocks(): Unit = {
           super.setupMocks()
           (mockPaymentsService.getOpenPayments(_: String)(_: HeaderCarrier, _: ExecutionContext))
             .expects(*, *, *)
             .returns(Future.successful(Some(Payments(Seq.empty))))
         }
+        private val result = target.openPayments()(fakeRequest)
+
+        status(result) shouldBe Status.OK
+      }
+    }
+
+    "the user is not authorised" should {
 
-        val result: Result = await(target.openPayments()(fakeRequest))
-        val document: Document = Jsoup.parse(bodyOf(result))
+      "return 403 (Forbidden)" in new Test {
+        override val authResult: Future[Nothing] = Future.failed(InsufficientEnrolments())
+        private val result = target.openPayments()(fakeRequest)
+        status(result) shouldBe Status.FORBIDDEN
+      }
+    }
+
+    "the user is not authenticated" should {
 
-        document.select("h1").first().text() shouldBe "What you owe"
+      "return 401 (Unauthorised)" in new Test {
+        override val authResult: Future[Nothing] = Future.failed(MissingBearerToken())
+        private val result = target.openPayments()(fakeRequest)
+        status(result) shouldBe Status.UNAUTHORIZED
       }
     }
 
     "an error occurs upstream" should {
 
-      "return the payments error view" in new Test {
+      "return 500 (Internal server error)" in new Test {
         override def setupMocks(): Unit = {
           super.setupMocks()
           (mockPaymentsService.getOpenPayments(_: String)(_: HeaderCarrier, _: ExecutionContext))
             .expects(*, *, *)
             .returns(Future.successful(None))
         }
+        private val result = target.openPayments()(fakeRequest)
 
-        val result: Result = await(target.openPayments()(fakeRequest))
-
-        val document: Document = Jsoup.parse(bodyOf(result))
-
-        document.select("h1").first().text() shouldBe "We can't let you pay here right now"
+        status(result) shouldBe Status.INTERNAL_SERVER_ERROR
```

Comment:
I feel like the test for the title of the view returned should be left in for these as if we're not testing it in integration tests, there's nothing to check we're rendering the correct view to go with the status code